### PR TITLE
Dsa

### PR DESCRIPTION
**Pull request by AnkitaGupta454**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes # <2454> 

<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] DSA

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by nimisha-yadav** [comment]
@AnkitaGupta454 remove the brackets and do not add space between # and digits then your issue will be attached.

**Comment by nimisha-yadav** [comment]
Please review @harshsri28 @sneha-nitdgp

File: DSA/Readme.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+## Next Greater Element
+<hr>
+
+```
+
+Given an array arr[ ] of size N having distinct elements, the task is to find the next greater element for each element of the array in order of their appearance in the array.
```
Comment (harshsri28) [inline DSA/Readme.md:6]: Add Problem statement  heading

File: DSA/Readme.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Input: 
+N = 4, arr[] = [1 3 2 4]
+Output:
+3 4 4 -1
+Explanation:
+In the array, the next larger element 
+to 1 is 3 , 3 is 4 , 2 is 4 and for 4 ? 
+since it doesn't exist, it is -1.
+
```
Comment (harshsri28) [inline DSA/Readme.md:24]: Discuss intuition and your approach and Time complexity and Space complexity

**Review by harshsri28 — CHANGES_REQUESTED** [review]
Add problem statement heading , intuition , algorithm and discuss about TC and SC

**Review by harshsri28 — APPROVED** [review]
LGTM

**Review by sneha-nitdgp — APPROVED** [review]
Good

**Review by nimisha-yadav — CHANGES_REQUESTED** [review]
@AnkitaGupta454 I'm sorry to say but 92% of your file content has been found plagiarized from online resources like GFG and Ideserve. I'm closing this PR. Kindly take up a new issue and from next time, research the topic and try to write in your own words. Also, do not forget to provide references for pictures used in your content.

**Pull request closed without merging**